kolibri/plugins/coach/assets/src/views/groups-page/index.vue

```diff
@@ -79,16 +79,16 @@
     },
     computed: {
       showCreateGroupModal() {
-        return this.modalShown === GroupModals.CREATE_GROUP;
+        return this.groupModalShown === GroupModals.CREATE_GROUP;
```

Comment:
> Couldn't we just have a method call an action to change the state?

I don't understand what you mean - we do have an action to change the state called `displayModal` in _actions/groups.js_.

This computed property should maybe be named differently - `createGroupModalIsShown` instead of `showCreateGroupModal`, which sounds like a command.

> Also not seeing the point of having an object represent the modals. We can't display more than one modal at a time, so every modal that isn't active would be false?

Again, not sure I understand. What object are you referring to? Here, `groupModalShown` is set to a constant string, not an object.

(For no modal shown, it looks like we set it to `false` but it should probably be `null` instead...)